cleancode/src/main/java/com/epam/cleancode/comments/MortgageInstallmentCalculator.java

```diff
@@ -2,38 +2,35 @@
 
 public class MortgageInstallmentCalculator {
 
+    private static final String MESSAGE_BAD_INPUT_PARAMETERS = "Negative values are not allowed";
+    private static final int AMOUNT_MONTHS_IN_YEAR = 12;
+
     /**
-     *
-     * @param p principal amount
-     * @param t term of mortgage in years
-     * @param r rate of interest
+     * @param startAmount principal amount
```

Comment:
comments is not necessary because parameters have friendly name and what is result also very clear